perf/src/packet.rs

```diff
@@ -226,6 +228,9 @@ pub fn to_packet_batches<T: Serialize>(items: &[T], chunk_size: usize) -> Vec<Pa
         .collect()
 }
 
+pub type BankingPacketBatch = Arc<Vec<PacketBatch>>;
+pub type BankingPacketReceiver = Receiver<BankingPacketBatch>;
```

Comment:
these types are planned to be by `solana-unified-scheduler-pool` crate, so they needed to be moved higher in the dep graph from `solana-core`.